### PR TITLE
Add w tagger

### PR DESCRIPTION
**Pull request by kratsg**
I can verify that JSSUtils compiles and runs fine and produces results equivalent to https://github.com/kratsg/BosonTaggerXAOD.

I haven't checked the Tool to see if it compiles, but I'm sure you can. I decided to `static_cast<int>(tagger.result(jet))` instead since booleans are problematic.

### CODE REVIEW

File: JetSubStructureMomentTools/Root/BosonTagTool.cxx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  JetSubStructureMomentToolsBase(name)
+{
+  declareProperty("WorkingPoint", m_working_point               = "medium");
+  declareProperty("TaggerAlgorithm", m_tagger_alg               = "tau21");
+  declareProperty("RecommendationsFile", m_recommendations_file = "$ROOTCOREBIN/data/JetSubStructureUtils/config_13TeV_20150306.dat");
+  declareProperty("ApplyMassCut", m_compare_mass                = true);
+  declareProperty("ApplyVariableCut", m_compare_variable        = true);
+  declareProperty("EnableSimpleTagger", m_simple_tagger         = false);
+  declareProperty("Debug", m_debug                              = false);
+}
```
Comment (kratsg) [inline JetSubStructureMomentTools/Root/BosonTagTool.cxx]: Defaults here are the same as https://github.com/kratsg/JetSubstructureTools/blob/addWTagger/JetSubStructureUtils/JetSubStructureUtils/BosonTag.h#L59-L65

File: JetSubStructureMomentTools/Root/BosonTagTool.cxx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+int BosonTagTool::modifyJet(xAOD::Jet &jet) const
+{
+  JetSubStructureUtils::BosonTag tagger(m_working_point, m_tagger_alg, m_recommendations_file, m_compare_mass, m_compare_variable, m_simple_tagger, m_debug);
+  jet.setAttribute("BosonTag", static_cast<int>(tagger.result(jet)));
+
```
Comment (kratsg) [inline JetSubStructureMomentTools/Root/BosonTagTool.cxx]: Could make it add multiple permutations such as

```c++
JetSubStructureUtils::BosonTag tagger_medium(m_working_point, m_tagger_alg, m_recommendations_file, m_compare_mass, m_compare_variable, m_simple_tagger, m_debug);
JetSubStructureUtils::BosonTag tagger_tight(m_working_point, m_tagger_alg, m_recommendations_file, m_compare_mass, m_compare_variable, m_simple_tagger, m_debug);

jet.setAttribute("BosonTag", static_cast<int>( (tagger_tight.result(jet)) << 1) | (tagger_medium.result(jet) << 0) );
```

**Comment by kratsg** [comment]
Need to update for non-rootcore with

```
#ifndef ROOTCORE
  path = PathResolverFindDataFile(…)
#endif
```

**Comment by mileswu** [comment]
Example from ApplyJetCalibration:

apply_pattern generic_declare_for_link kind=xmls files='-s=../data/InsituCalibration *.root' prefix=XML/ApplyJetCalibration/InsituCalibration name=InsituCalibration

#include "PathResolver/PathResolver.h"
std::string fullpath = PathResolverFindXMLFile("ApplyJetCalibration/stuff");

**Comment by kratsg** [comment]
@mileswu New commit that should add access to the files without `$ROOTCOREBIN` being defined.

**Comment by kratsg** [comment]
```
source /cvmfs/atlas.cern.ch/repo/ATLASLocalRootBase/x86_64/AtlasSetup/current/AtlasSetup/scripts/asetup.sh devval,rel_0,gcc48,64,opt,here --nightliesarea=/cvmfs/atlas-nightlies.cern.ch/repo/sw/nightlies/x86_64-slc6-gcc48-opt
```

then create a directory `Reconstruction/Jet` and symlink all JSSU into it, then

```
cd Reconstruction/Jet/JetSubStructureUtils/cmt; cmt config; make; cd $TestArea   
cd Reconstruction/Jet/JetSubStructureMomentTools/cmt; cmt config; make; cd $TestArea
```

**Comment by kratsg** [comment]
Rebased, This should be :gun: to :ok:

**Pull request merged**